apollo/lib/exceptions/__init__.py

```diff
@@ -19,6 +20,12 @@ def __init__(self, message, location, schemaObject, *args, **kwargs):
         )
 
 
+class RequestValidationErrorWrapper(RequestValidationError):
```

Comment:
I'd just call this `RequestValidationError` and import the original one under an alias. It's weird for a caller to raise an exception that calls itself a wrapper.